### PR TITLE
Added L8 training files

### PR DESCRIPTION
**Pull request by shradhasehgal**

### CODE REVIEW

File: torchgeo/datasets/l8biome.py [hunk 1]
```diff
@@ -206,7 +206,7 @@ def __getitem__(self, query: BoundingBox) -> dict[str, Any]:
 
         mask_filepaths = []
         for filepath in filepaths:
-            mask_filepath = filepath.replace("B1.TIF", "fixedmask.img")
+            mask_filepath = filepath.replace("B1.TIF", "fixedmask.tif")
```
Comment (adamjstewart) [inline torchgeo/datasets/l8biome.py:209]: Remind me to reupload the new version of the dataset.

File: experiments/run_l8biome_experiments.py [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        if not os.path.exists(os.path.join(output_dir, experiment_name)):
+            command = (
+                "python train.py"
+                + f" config_file={config_file}"
+                + f" experiment.name={experiment_name}"
+                + f" experiment.module.segmentation_model={model}"
+                + f" experiment.module.learning_rate={lr}"
+                + f" experiment.module.loss={loss}"
+                + f" experiment.module.backbone={backbone}"
+                + f" experiment.module.weights={weight_init}"
```
Comment (adamjstewart) [inline experiments/run_l8biome_experiments.py:55]: There is no experiment key anymore, so all of these keys are being ignored. They should be module and datamodule instead

**Comment by adamjstewart** [comment]
What's the status of this PR?

**Comment by shradhasehgal** [comment]
> What's the status of this PR?

Can be closed

**Pull request closed without merging**